go/libkb/api.go

```diff
@@ -189,7 +189,13 @@ func doRequestShared(api Requester, arg APIArg, req *http.Request, wantJSONRes b
 		return
 	}
 
+	dbg := func(s string) {
```

Comment:
on the off chance that GetTorMode() is the culprit, can we log entry into this function?